recipes/sqlpp11/all/test_package/CMakeLists.txt

```diff
@@ -7,5 +7,5 @@ conan_basic_setup(TARGETS)
 find_package(Sqlpp11 CONFIG REQUIRED)
 
 add_executable(example example.cpp)
-set_property(TARGET example PROPERTY CXX_STANDARD 11)
+set_property(TARGET example PROPERTY CXX_STANDARD 14)
```

Comment:
Why do you need C++14 now? I didn't find any information on https://github.com/rbock/sqlpp11/releases/tag/0.61

Reply:
I was getting the following error: `error: constexpr function's return type 'void' is not a literal type`

https://github.com/conan-io/conan-center-index/pull/8783

Reply:
They added the `constexpr void` in 0.61: https://github.com/rbock/sqlpp11/compare/0.60...0.61#diff-5341f4f4a83948ea9eee5499ef9ad879c329456f997156233974608d1dd7bf6eR37

Reply:
Excellent, in this case, you need to add a restriction in the recipe too. The test package package can be prepared now, but users won't. I'll open a PR to you asap.

Reply:
@sonicxml consider https://github.com/sonicxml/conan-center-index/pull/1

Reply:
Merged, thanks!